src/app/3d/qgsmesh3dsymbolwidget.cpp

```diff
@@ -111,6 +115,23 @@ void QgsMesh3dSymbolWidget::setLayer( QgsMeshLayer *meshLayer, bool updateSymbol
 {
   mLayer = meshLayer;
 
+  mComboBoxTriangleCount->clear();
+  if ( meshLayer && meshLayer->meshSimplificationSettings().isEnabled() )
+  {
+    mComboBoxTriangleCount->setVisible( true );
+    mLabelTriangleCount->setVisible( true );
+    int lodCount = meshLayer->triangularMeshLevelOfDetailCount();
+    for ( int i = 0; i < lodCount; ++i )
+    {
+      mComboBoxTriangleCount->addItem( QString::number( meshLayer->triangularMeshByLodIndex( i )->triangles().count() ) );
```

Comment:
To make this a bit clearer for users, I'd suggest that the combo box strings include the level too. E.g. instead of just the raw count "1234134", maybe something like

    Level 1 (123342 triangles)
    Level 2 (5453 triangles)

etc

Or, alternatively, if the level of detail count is calculated by just taking a fixed % of triangles, then expose that here:

   "100% (234234 triangles)"
   "75% (123123 triangles)"

etc

Reply:
this:
![combo](https://user-images.githubusercontent.com/7416892/95703055-20922b80-0c1c-11eb-91d8-15414aa1fd82.gif)
?

Reply:
Whatt's the 1/10, 1/112  etc represent?

Reply:
the reduction factor from the original mesh. Percentage is not pertinent because the reduction is too strong.

Reply:
So 1/10 = 10% of the triangles?

Reply:
> So 1/10 = 10% of the triangles?

yes, that's it

Reply:
Hmm that looks a bit odd to me still. @nirvn have you any ideas here?

Reply:
IMHO, just "XXX triangles" is enough...

Reply:
I'm also not a fan of the combobox with XXXX triangles.

What about using a slider instead? I feel that could do a better UX job. 

Each step in the slider would represent 1/10, 1/137, etc. Users would slide left/right to obtain right level of simplification.

It'd behave very much like the shapebuster fill symbol layer's quality slider.

Reply:
![slider gif](https://user-images.githubusercontent.com/7416892/95804436-acfe2600-0cd0-11eb-9cc8-83cda2423659.gif)

Reply:
Much better IMHO.